packages/bot-web-ui/src/components/dashboard/bot-builder/bot-builder.tsx

```diff
@@ -83,21 +83,11 @@ const BotBuilder = ({
                         <TourSlider />
                     </MobileWrapper>
                     <DesktopWrapper>
-                        <ReactJoyride
+                        <ReactJoyrideWrapper
                             steps={BOT_BUILDER_TOUR}
                             run={is_tour_running}
-                            continuous
                             showProgress
-                            callback={handleJoyrideCallback}
-                            locale={{ back: 'Previous' }}
                             styles={{
-                                options: {
-                                    arrowColor: 'var(--general-main-2)',
-                                    backgroundColor: 'var(--general-main-2)',
-                                    primaryColor: 'var(--brand-red-coral)',
-                                    textColor: 'var(--text-general)',
-                                    spotlightShadow: '0 0 15px rgba(0, 0, 0, 0.5)',
-                                },
                                 buttonBack: {
```

Comment:
@vinu-deriv can we remove `buttonBack` from this file as it is already in ReactJoyrideWrapper?

Reply:
@vinu-deriv In the testlink it is actually using `border: 0.2rem` for both the tour. And i think we should continue to use 0.2rem for both to keep it consistent. I believe this was the only difference in styles. if thats the case then lets remove the buttonBack from here.